rust_backend/Cargo.toml

```diff
@@ -0,0 +1,27 @@
+[package]
+name = "rust_backend"
+version = "0.1.0"
+edition = "2021"
+
+[lib]
+crate-type = ["cdylib", "rlib"]
+
+[dependencies]
```

Comment:
There are lots of boilerplatey dependencies, but I think all are worth it for a decent developer experience